frappe/public/js/frappe/views/kanban/kanban_card.html

```diff
@@ -1,9 +1,11 @@
 <div class="kanban-card-wrapper {{ disable_click }}" data-name="{{name}}">
-	<div class="kanban-card content">
-		<div class="kanban-card-title">
-			{{ title }}
+	<a class="kanban-card-redirect" href="#" target="_blank">
```

Comment:
```suggestion
	<a class="kanban-card-redirect" href="#">
```

adding `target="_blank"` still forces the user to open it in new tab. 

The idea here is to let use choose if the user wants to open it in new tab. If the user wants he/she can open it in new tab by ctrl + clicking it.

Reply:
@surajshetty3416 target attribute is removed in the latest commit.